text/entities/0256-entities-data-model.md

```diff
@@ -0,0 +1,790 @@
+identification context. Note: GID(E) is a map of key/value attributes.
+
+The enrichment process often is responsible for determining the value of `IDCONTEXT(E)`
+and for computing the GID according to the formula defined above, although the GID may
+also be produced at once by the telemetry source (e.g. by OTel SDK) without requiring
+any additional enrichment.
+
+### Semantic Conventions
+
+OpenTelemetry semantic conventions will be enhanced to include entity definitions for
+well-known entities such as Service, Process, Host, etc.
+
+For well-known entity types LID(E) is defined in OTel semantic conventions per
+entity type. The value of LID is a map of key/value attributes. For example,
+for entity of type "process" the semantic conventions define LID as 2 attributes:
+
+```json5
+{
+  "process.pid": $pid,
+  "process.start_time": $starttime
+}
+```
+
+For custom entity types (not defined in OTel semantic conventions) the end-user is
+responsible for defining their custom semantic conventions in a similar way.
+
+The entity information producer is responsible for determining the identification
+context of each entity it is producing information about.
+
+In certain cases, where only one possible IDCONTEXT definition is meaningful, the
+IDCONTEXT may be defined in the semantic conventions. For example Kubernetes nodes
+always exist in the identifying context of a Kubernetes cluster. The semantic convention
+for "k8s.node" and "k8s.cluster" can prescribe that the IDCONTEXT of entity of type
+"k8s.node" is always an entity of type "k8s.cluster".
+
+Important: semantic conventions are not expected to (and normally won't) prescribe
+the complete GID composition.
+Semantic conventions should prescribe LID and may prescribe IDCONTEXT, but GID
+composition, generally speaking, cannot be known statically.
+
+For example: a host's LID should be a `host.id` attribute. A host running on a cloud
+should have an IDCONTEXT of "cloud.account" and the LID of "cloud.account" entity
+is (`cloud.provider`, `cloud.account.id`). However semantic conventions cannot prescribe
+that the GID of a host is (`host.id`, `cloud.provider`, `cloud.account.id`) because not all
+hosts run on cloud. A host that runs on prem in a single data center may have a GID
+of just (`host.id`) or if a customer has multiple on prem data centers they may use
+data.center.id as its identifier and use (`host.id`, `data.center.id`) as GID of the host.
+
+### Examples
+
+_This section is a supplementary guideline and is not part of logical data model._
+
+#### Process in a Host
+
+A locally running host agent (e.g. an OTel Collector) that produces
+information about "process" entities has the knowledge that the
+processes run in the particular host and thus the "host" is the
+identification context for the processes that the agent observes. The
+LID of a process can look like this:
+
+```json5
+{
+  "process.pid": 12345,
```

Comment:
Hi @tigrannajaryan, I know this is only an example, but I'd like to point out this won't be sufficient for a unique process entity ID on Windows.  

On Windows, PIDS can be re-used in less than 1s, so to be unique, the start time could be more precise (nanoseconds), or more attributes are needed

Reply:
Yes, just an example and thanks for pointing out. We will work on this most likely within the semconv group when the Entities SIG makes a bit more progress on the basics.